src/main/java/seedu/mtracker/commands/ListCommand.java

```diff
@@ -1,11 +1,22 @@
 package seedu.mtracker.commands;
 
+import seedu.mtracker.instrument.Instrument;
+import seedu.mtracker.ui.TextUi;
+
+import java.util.ArrayList;
+
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
+    private ArrayList<Instrument> instruments;
+
     @Override
     public String execute() {
+        instruments = instrumentManager.getInstruments();
+        for (Instrument i : instruments) {
+            TextUi.displayInstruments(i);
+        }

```

Comment:
Can shift this as a function into TextUi, including the for loop, like TextUi.displayAllInstruments(instruments)

Try to use the Array.stream().forEach